ingest/annotations.py

```diff
@@ -79,7 +79,7 @@ def preproccess(self):
             "numeric", axis=1, level=1, drop_level=False
         ).columns.tolist()
         # TODO perform replace
-        self.file[numeric_columns] = self.file[numeric_columns].round(3).astype(float)
+        self.file[numeric_columns] = self.file[numeric_columns].round(3).astype(int)
```

Comment:
How come we're casting it as an int?  Shouldn't it be a float?

Reply:
my bad. Had issues loading int64 and this was apart of the debugging process. Fixed in cbb6a24